terralab/utils.py

```diff
@@ -33,14 +33,26 @@ def wrapper(*args: str, **kwargs: str) -> Any:
                 try:
                     message = json.loads(e.body)["message"]
                 except json.JSONDecodeError:
-                    message = e.body
+                    try:
+                        message = (
+                            e.reason
+                        )  # 401 Unauthorized stores Unauthorized in Reason
```

Comment:
wtf.  is this how 401 unauthorized is normally handled or is this like a  "our service/TCL" thing?

also what is in `e.body` for these 401 unauthorized responses?

Reply:
e.body is the HTML doc 🙃 
```
HTTP response body: <!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML 2.0//EN">
<html><head>
<title>401 Unauthorized</title>
</head><body>
<h1>Unauthorized</h1>
<p>This server could not verify that you
are authorized to access the document
requested.  Either you supplied the wrong
credentials (e.g., bad password), or your
browser doesn't understand how to supply
the credentials required.</p>
</body></html>
```

Reply:
so alternatively I could remove the whole e.reason logic and instead make the if statement 
`elif e.status == 401 and "401 Unauthorized" in message:`